db-migrations/src/Database/Migration/CLI.hs

```diff
@@ -63,11 +65,11 @@ parserInfo = wrapHelper commands "migration commands"
       <> mkCommand
          "dump-schema"
          (\Config{..} -> dumpSchema runPGDump)
-         ("Dump database schema to " <> schemaFileString)
+         "Dump database schema"
```

Comment:
@epicallan Probably not blocking, but I was curious if leaving out the path to the dump (which would now include the `migrationDir`, was done on purpose, or if you had perhaps planned to add it back at some point?

Reply:
@epicallan Aha, thanks for the explanation. Makes sense to not block on something minor like this.